src/APIFunctions/GoogleSheets.js

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios';
+import { ApiResponse } from './ApiResponses';
+import { MAILER_API_URL } from '../config/config.json';
+
+/**
+ * Invokes the google spreadsheet API to add officer application form data to it
+ * @param {object} row - the data from the officer application form
+ * @param {String} sheetsID - ID of the google sheet getting updated
+ * @returns {ApiResponse} Containing any error information related to the
+ * request
+ */
+export async function addToSpreadsheet(row, sheetsID){
```

Comment:
nitpicking, but so its camel case do:
```suggestion
export async function addToSpreadsheet(row, sheetsId){
```

change the associated comment above too